navigation/navigation-fragment/src/main/java/androidx/navigation/fragment/FoldableFragmentNavigator.java

```diff
@@ -214,6 +217,7 @@ public FoldableNavDestination navigate(@NonNull Destination destination, @Nullab
         // The commit succeeded, update our view of the world
         if (isAdded) {
             mBackStack.add(destId);
+            Log.d("ANDREI", "BackStackSize = " + mBackStack.size());
```

Comment:
Do we need to keep this log or can it be removed?

Reply:
wops! nice catch!

Reply:
No. My mistake. 😆